promptsource/templates/dynabench/dynasent/dynabench.dynasent.r2.all/templates.yaml

```diff
@@ -0,0 +1,93 @@
+dataset: dynabench/dynasent
+subset: dynabench.dynasent.r2.all
+templates:
+  0514a97d-0bbc-4f78-bcaa-622f2898fd8b: !Template
+    answer_choices: negative ||| positive ||| neutral ||| mixed
+    id: 0514a97d-0bbc-4f78-bcaa-622f2898fd8b
+    jinja: 'I''m reading a review that says "{{sentence}}".
+
+
+      Do you think the review is {{"positive"}} or {{"negative"}} or {{"neutral"}}
+      or {{"mixed"}}? ||| {{gold_label}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: review
+    reference: ''
+  4acb117e-e2f5-46a1-9cd5-17d9d175c1e4: !Template
+    answer_choices: bad ||| good ||| neutral ||| mixed
+    id: 4acb117e-e2f5-46a1-9cd5-17d9d175c1e4
+    jinja: 'Someone sent me an email with the sentence "{{sentence}}". Do you think
+      they are feeling {{"good"}} or {{"bad"}} or {{"neutral"}} or {{"mixed"}}?
+
+      |||
+
+      {% if gold_label == "positive" %}
+
+      good
+
+      {% elif gold_label == "negative" %}
+
+      bad
+
+      {% else %}
+
+      {{gold_label}}
+
+      {% endif %}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: happy or mad
+    reference: ''
+  51534992-1a8a-42c7-9d94-fd330942cbf5: !Template
+    answer_choices: negative ||| positive ||| neutral ||| mixed
+    id: 51534992-1a8a-42c7-9d94-fd330942cbf5
+    jinja: 'Does the following sentence have a {{"positive"}} or {{"negative"}} or
```

Comment:
order of responses in the answer choices doesn't match order of possible answers in the prompt. Please order so they are in the same order.